concepticondata/conceptlists/Dawson-1887-731.tsv

```diff
@@ -0,0 +1,732 @@
+Dawson-1887-731-60	60	Shoulder-blade	172	SHOULDERBLADE	Part of the Body	2	28
+Dawson-1887-731-61	61	Back	1291	BACK	Part of the Body	2	28
+Dawson-1887-731-62	62	Breast	1402	BREAST	Part of the Body	2	28
+Dawson-1887-731-63	63	Hip	1745	HIP	Part of the Body	2	28
+Dawson-1887-731-64	64	Belly	1251	BELLY	Part of the Body	2	28
+Dawson-1887-731-65	65	Navel	1838	NAVEL	Part of the Body	2	28
+Dawson-1887-731-66	66	Arm	1673	ARM	Part of the Body	2	28
+Dawson-1887-731-67	67	Right arm			Part of the Body	2	28
+Dawson-1887-731-68	68	Left arm			Part of the Body	2	28
+Dawson-1887-731-69	69	Arm pits	1886	ARMPIT	Part of the Body	2	28
+Dawson-1887-731-70	70	Elbow	981	ELBOW	Part of the Body	2	28
+Dawson-1887-731-71	71	Wrist	799	WRIST	Part of the Body	2	28
+Dawson-1887-731-72	72	Hand	1277	HAND	Part of the Body	2	28
+Dawson-1887-731-73	73	Palm of hand	1183	PALM OF HAND	Part of the Body	2	28
+Dawson-1887-731-74	74	Back of hand			Part of the Body	2	28
+Dawson-1887-731-75	75	Fingers	1303	FINGER	Part of the Body	2	28
+Dawson-1887-731-76	76	Thumb	1781	THUMB	Part of the Body	2	28
+Dawson-1887-731-77	77	First finger			Part of the Body	2	28
+Dawson-1887-731-78	78	Second finger	3387	MIDDLE FINGER	Part of the Body	2	28
+Dawson-1887-731-79	79	Third finger			Part of the Body	2	28
+Dawson-1887-731-80	80	Small finger			Part of the Body	2	28
+Dawson-1887-731-81	81	Finger nail	1258	FINGERNAIL	Part of the Body	2	28
+Dawson-1887-731-82	82	Knuckle	2824	KNUCKLE	Part of the Body	2	28
+Dawson-1887-731-83	83	Space between knuckles			Part of the Body	2	28
+Dawson-1887-731-84	84	Rump			Part of the Body	2	28
+Dawson-1887-731-85	85	Leg	1297	LEG	Part of the Body	2	28
+Dawson-1887-731-86	86	Leg above knee	471	UPPER LEG	Part of the Body	2	28
+Dawson-1887-731-87	87	Knee	1371	KNEE	Part of the Body	2	28
+Dawson-1887-731-88	88	Leg below knee	447	LOWER LEG	Part of the Body	2	28
+Dawson-1887-731-89	89	Calf of the leg	498	CALF OF LEG	Part of the Body	2	28
+Dawson-1887-731-90	90	Ankle	803	ANKLE	Part of the Body	2	29
+Dawson-1887-731-91	91	Ankle bone			Part of the Body	2	29
+Dawson-1887-731-92	92	Instep	3736	INSTEP	Part of the Body	2	29
+Dawson-1887-731-93	93	Foot	1301	FOOT	Part of the Body	2	29
+Dawson-1887-731-94	94	Sole of foot	2666	SOLE (FOOT)	Part of the Body	2	29
+Dawson-1887-731-95	95	Heel	980	HEEL	Part of the Body	2	29
+Dawson-1887-731-96	96	Toes	1389	TOE	Part of the Body	2	29
+Dawson-1887-731-97	97	Large toe			Part of the Body	2	29
+Dawson-1887-731-98	98	Second toe			Part of the Body	2	29
+Dawson-1887-731-99	99	Third toe			Part of the Body	2	29
+Dawson-1887-731-100	100	Toe nail	2662	TOENAIL	Part of the Body	2	29
+Dawson-1887-731-101	101	Blood	946	BLOOD	Part of the Body	2	29
+Dawson-1887-731-102	102	Vein or artery	525	BLOOD VESSEL	Part of the Body	2	29
+Dawson-1887-731-103	103	Brain	1583	BRAIN	Part of the Body	2	29
+Dawson-1887-731-104	104	Bladder	1746	BLADDER	Part of the Body	2	29
+Dawson-1887-731-105	105	Gall	1780	BILE	Part of the Body	2	29
```

Comment:
I think this rather refers to the gall bladder, not the substance stored within.

Reply:
The Collins dictionary says it is an obsolete term for 'bile', which may be the case considering the date of the list publication. Also there are other instances of 'gall' (see Swadesh-1950-165-136, Smith-2015-97-48) linked to BILE.